src/components/GiftRanking/FilterGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import * as S from './GiftRankingSection.styles';
+
+interface FilterGroupProps {
+  items: string[];
+  selected: string;
```

Comment:
```suggestion
  value: string;
```
이렇게 표현해도 괜찮을 것 같네요..!?